source/plugins/quota.rst

```diff
@@ -0,0 +1,189 @@
+governing permissions and limitations under the License.
+
+
+Quota Plugin 
+=============
+
+Quota service, while allowing for scalability, will make sure that the cloud is
+not exploited by attacks, careless use and program errors. To address this
+problem, we propose to employ a quota-enforcement service that allows resource
+usage within certain bounds as defined by policies and available quotas for
+various entities. Quota service extends the functionality of usage server to
+provide a measurement for the resources used by the accounts and domains using a
+common unit referred to as cloud currency in this document. It can be configured
+to ensure that your usage won’t exceed the budget allocated to accounts/domain
+in cloud currency. It will let user know how much of the cloud resources he is
+using. It will help the cloud admins, if they want, to ensure that a user does
+not go beyond his allocated quota. Per usage cycle if a account is found to be
+exceeding its quota then it is locked. Locking an account means that it will not
+be able to initiate a new resource allocation request, whether it is more
+storage or an additional ip. Needless to say quota service as well as any action
+on the account is configurable.
+
+Enabling the Quota Service 
+----------------------------
+
+Before installing and configuring the quota service you need to make sure that
+the Usage Server has been installed. This requires extra steps beyond just
+installing the CloudStack software. See Installing the Usage Server (Optional)
+in the Advanced Installation Guide.
+
+#. enable.usage.server: Set to true to enable usage server.
+
+The quota plugin is disabled by default. To enable it goto Global Settings and
+set the following global configuration to true:
+
+#.  quota.enable.service
+
+By default Quota service does not lock the accounts that have exceeded the quota
+usage. To enable quota service to lock accounts set the following global
+configuration to true:
+
+#. quota.enable.enforcement
+
+The other configurations that are there for quota service are as:
+
+#. quota.currency.symbol : The symbal that is used before any currency 
+figure in various quota forms and reports. 
+#. quota.usage.smtp.host: Quota SMTP host for sending quota alerts. 
+#. quota.usage.smtp.port: Quota SMTP port. 
+#. quota.usage.smtp.user: Quota SMTP user. 
+#. quota.usage.smtp.password: Quota SMTP password. 
+#. quota.usage.smtp.sender: Quota SMTP alert sender email address. 
+#. quota.usage.smtp.useAuth: If true, use secure SMTP authentication when sending
+	emails. 
+#. quota.usage.smtp.connection.timeout: Quota SMTP server connection
+	timeout duration.
+
+There are several configuration variables that are inherited from usage server.
+These are listed below:
+
+#. usage.aggregation.timezone 
+
+All these are described in details in Usage Server documentation.
+
+Restart the Management Server (as usual with any global configuration change)
```

Comment:
usual but not always needed. I would rephrase to (to enable the set configuration values)